adapter/prometheus/prometheus_test.go

```diff
@@ -69,7 +69,7 @@ var (
 		Name:        "unknown",
 		Description: "unknown",
 		Kind:        adapter.Gauge - 2,
```

Comment:
Given we use Kind in config to reference aspects, maybe we should rename this Kind field to Flavor or somesuch?